packages/react-components/package.json

```diff
@@ -99,6 +99,7 @@
     "@iot-app-kit/core-util": "7.1.0",
     "@iot-app-kit/source-iottwinmaker": "7.1.0",
     "color": "^4.2.3",
+    "copy-to-clipboard": "^3.3.3",
```

Comment:
https://github.com/sudodoki/copy-to-clipboard uses a deprecated API and hasn't been updated in a year. Might want to consider https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Interact_with_the_clipboard#writing_to_the_clipboard.

Reply:
given the number of usages / downloads, i assumed it to be ok to use... https://www.npmjs.com/package/copy-to-clipboard